scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -24,3 +25,8 @@ def printStartDateStamp():
 def printIndented( text, indentLevel = 1 ):
   for line in text.splitlines():
     print( ( '\t' * indentLevel ) + line )
+
+if __name__ == "__main__":
```

Comment:
Yeah, strictly speaking this script would never be directly called.. But I like the idea of having it for unit testing.

And yeah, if you want to show me some cool testing tricks, I'd love it!